src/test/java/seedu/fast/testutil/TypicalPersons.java

```diff
@@ -224,6 +224,21 @@ public static Fast getTypicalFastSetTwo() {
         return fast;
     }
 
+    /**
+     * Returns an {@code Fast} with all the typical persons.
+     */
+    public static Fast getTypicalFastSetThree() {
+        Fast fast = new Fast();
+        for (Person person : getTypicalPersons()) {
+            fast.addPerson(person);
+        }
+
+        for (Person person : getTypicalPersonsSetTwo()) {
+            fast.addPerson(person);
+        }
+        return fast;
+    }
+
     public static List<Person> getTypicalPersons() {
```

Comment:
Just a sidenote:
Might need to update all our testcases in the future, so that all testcases use the same list of typical persons instead of needing updated lists for newer testcases.